changelog/unreleased/issue-12555.toml

```diff
@@ -0,0 +1,5 @@
+type = "a"
+message = "Generate an event in the `System Event` stream for every system notification"
+
```

Comment:
This could have some more details in the `details.user` section. Like what can be done with it

Reply:
Good point - added more detail